odoo/addons/base/data/ir_module_category_data.xml

```diff
@@ -22,7 +22,7 @@
             <field name="visible" eval="0" />
         </record>
 
-        <record model="ir.module.category" id="module_category_localization_account_charts">
+        <record model="ir.module.category" id="module_category_accounting_localizations_account_charts">
             <field name="parent_id" ref="module_category_accounting_localizations" />
             <field name="name">Account Charts</field>
```

Comment:
The name Account Charts is weird to me as everywhere else, we are talking about Chart Templates or Chart of Accounts (CoA)

Reply:
We are talking about a category that correspond to "COA + taxes + fiscal position", renaming it "Chart of accounts" is not entirely correct in my opinion as it is more than just the chart of accounts. Most localization module description are using the term "Accounting Charts", maybe we can use the same name which might be a bit clearer.

Reply:
@csnauwaert Or Chart Template then?  Would not the user see it as taxes/fps are integrated with the CoA?

Reply:
![image](https://user-images.githubusercontent.com/44770049/89538245-a0ee7780-d7fa-11ea-8360-29760a265cd7.png)
Or fiscal localization?

Reply:
if we follow that logic, it is 'package'